includes/class-options.php

```diff
@@ -121,4 +125,55 @@ public static function maybe_disable_interactions( $pre_option ) {
 
 		return $pre_option;
 	}
+
+	/**
+	 * Default option filter for the Vary Header.
+	 *
+	 * @see https://github.com/Automattic/wordpress-activitypub/wiki/Caching
+	 *
+	 * @param string $default The default value of the option.
+	 *
+	 * @return string The default value of the option.
+	 */
+	public static function default_option_activitypub_vary_header( $default ) {
```

Comment:
This might be obsolete if we decide to enable the `Vary` header by default #1638